dash/dash.py

```diff
@@ -328,7 +329,8 @@ def serve_layout(self):
     def _config(self):
         config = {
             'url_base_pathname': self.url_base_pathname,
-            'requests_pathname_prefix': self.config['requests_pathname_prefix']
+            'requests_pathname_prefix': self.config['requests_pathname_prefix'],  # noqa: E501
```

Comment:
Could also use the dot notation for less characters: `self.config.requests_pathname_prefix`.

Reply:
Oh interesting, didn't realize we had a js-like dict class `AttributeDict`. FWIW that came in 18 months ago: https://github.com/plotly/dash/commit/885340256a92d2242c622a8e57fe7064890e80cb for backward compatibility with an old class-based config. Honestly I find that a bit confusing and un-pythonic, has it been long enough that we could get rid of the dot version?

Would also be nice to remove the spelling mistake fallback `supress_callback_exceptions` and just throw an error if someone tries to use the misspelled version. Maybe even list the allowed keys explicitly and throw a helpful error if any other key is provided.

Reply:
I used this library `attrdict` before, and mostly the use case is for config.  yes we should create an issue for supress typo if there is no one yet
https://pypi.org/project/attrdict/